test/verify/check-storage-mounting

```diff
@@ -140,5 +140,54 @@ class TestStorage(StorageCase):
         self.dialog({ "type": "empty" })
         self.content_row_wait_in_col(1, 1, "Unrecognized Data")
 
+    def testMountOptions(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/storage")
+
+        # Add a disk
+        m.add_disk("50M", serial="MYDISK")
+        b.wait_in_text("#drives", "MYDISK")
+        b.click('#drives tr:contains("MYDISK")')
+        b.wait_visible('#storage-detail')
+
+        # Open format dialog and play with the checkboxes
+
+        self.content_tab_action(1, 1, "Format")
+        self.dialog_wait_open()
+        self.dialog_set_val("type", "luks+ext4")
+        self.dialog_set_val("mounting", "custom")
+
+        def wait_checked(field):
+            b.wait_present(self.dialog_field(field) + ":checked")
+
+        def wait_not_checked(field):
+            b.wait_present(self.dialog_field(field) + ":not(:checked)")
+
+        wait_checked("crypto_options_auto")
+        wait_checked("mount_auto")
+        wait_not_checked("crypto_options_ro")
+        wait_not_checked("mount_ro")
+
+        # Uncheck crypto auto.  This gets propagated to mount auto.
+        self.dialog_set_val("crypto_options_auto", False)
+        self.dialog_wait_val("crypto_options", "noauto")
+        self.dialog_wait_val("mount_options", "noauto")
+        wait_not_checked("mount_auto")
+
+        # Check crypto ro.  This gets propagated to ount ro.
+        self.dialog_set_val("crypto_options_ro", True)
+        self.dialog_wait_val("crypto_options", "noauto,readonly")
+        self.dialog_wait_val("mount_options", "noauto,ro")
+        wait_checked("mount_ro")
+
+        # Set mount options string.
+        self.dialog_set_val("mount_options", "foo")
+        wait_checked("mount_auto")
+        wait_not_checked("mount_ro")
+        self.dialog_set_val("mount_options", "foo,noauto")
+        wait_not_checked("mount_auto")
```

Comment:
Could this test also verify that the mount options actually get propagated all the way to `fstab`/`crypttab`? Or does this happen in a different test already?

Reply:
Good point.  We only check this for crypto options, but not for mount options.